keentools_facebuilder/utils/materials.py

```diff
@@ -237,10 +237,39 @@ def progress_callback(progress):
         bpy.context.window_manager.progress_update(progress)
         return False
 
-    bpy.context.window_manager.progress_begin(0, 1)
-    built_texture = tb.build_texture(
-        frames_count, frame_data_loader, progress_callback)
-    bpy.context.window_manager.progress_end()
+    #-------------- New baking
+    # bpy.context.window_manager.progress_begin(0, 1)
+    # built_texture = tb.build_texture(
+    #     frames_count, frame_data_loader, progress_callback)
+    # bpy.context.window_manager.progress_end()
+
+    #-------------- Old baking
```

Comment:
Why is that change in pull request? frame_data_loader should be changed instead

Reply:
We have no new texture builder in pykeentools within this branch. So after a successful merge, we just need to delete the old baking and uncomment the new one.